node/config.go

```diff
@@ -485,7 +459,7 @@ func getKeyStoreDir(conf *Config) (string, bool, error) {
 
 var warnLock sync.Mutex
 
-func (c *Config) warnOnce(w *bool, format string, args ...interface{}) {
+func (c *Config) warnOnce(w *bool, err bool, format string, args ...interface{}) {
```

Comment:
IMO this seems like a micro-customization. Is it that big of a deal if this method always does WARN instead of ERR ? 

I can live with the way it is though, just think it'd be cleaner to leave it be